src/multiconn-client.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import socket
```

Comment:
Earlier discussion agreed on HTTP & REST for communication. Was socket easier to use? We should have a rational justification for the RPC method chosen. for REST it was simplicity.